pkg/virt-handler/device-manager/usb_device_test.go

```diff
@@ -216,6 +216,13 @@ var _ = Describe("USB Device", func() {
 			},
 		),
 	)
+	BeforeEach(func() {
```

Comment:
This will break or impact the other tests, won't it? Can you set the unexported var in the test case `It` instead?

Reply:
It won't since the other test does not call `discoverPluggedUSBDevices()`. As for having it this way, please see this thread https://github.com/kubevirt/kubevirt/pull/11047#discussion_r1458902609

Reply:
I don't think we should taint other test cases, even though they might not use this variable right now.

Reply:
Then it is back to the question, use a `DeferCleanup` or drop the `BeforeEach` and do it inside the test?

Reply:
IMO inside the test is fine. Will that permanently alter the variable for subsequent runs of tests? (With BeforeEach it does too, but at least it's obvious.)

Reply:
With normal go test it should, since the test is running in the same process. For ginkgo i don't know, but i think it uses go test somehow so likely it is the same here.

Reply:
I have dropped the framework specific `BeforeEach`, because it is intended for repeating setup operations, whereas in this instance we only really want the variable to be changed for a single test. It is not shared code and should therefore not happen outside of the actual test function.